components/net_traits/lib.rs

```diff
@@ -137,6 +137,7 @@ pub struct LoadData {
     pub data: Option<Vec<u8>>,
     pub cors: Option<ResourceCORSData>,
     pub pipeline_id: Option<PipelineId>,
+    pub include_third_party_credentials: bool,
```

Comment:
I think this is called "credentials flag" in the spec? Fine with it being named like this, but we should have a doc link to the spec flag it corresponds to.